CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+16.1.0
+------
+* Use Session Token auth strategy by default for new embedded apps #1111
+* Create optional `EnsureAuthenticatedLinks` concern to authenticate deep links using Turbolinks #1118
```

Comment:
Let's link to the PRs as per the other entries?

Reply:
Done!